performance/get_time.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const { VASTClient } = require('../dist/vast-client-node.min.js');
+
+const url = path.join('file://', __dirname, '..', 'spec', 'samples', 'inline-linear.xml');
+
+const VastClient = new VASTClient();
+
+const timeBefore = Date.now();
```

Comment:
perhaps you could use `process.hrtime()` although it's more complex
https://nodejs.org/api/process.html#process_process_hrtime_time

Reply:
done